### ISSUE TITLE
[FEATURE] Limit quiz participations per user

### ISSUE THREAD

**Issue by gckng** [bihor/fp_masterquiz#56]
Hi there,
right now it is possible to check if the user has already participated.
There is a use case in one of our quizzes, where the user should have 3 possibilities to pass the quiz.
Maybe this could be useful for other users as well?

**Comment by gckng** [comment]
Thanks for your quick response.

Need this feature for logged in fe_users.
Like an extended feUserCheck, right now the quiz is feasible only once if the check is activated. 
Would be nice to have more tries (maybe set by an integer value).

**Comment by bihor** [comment]
OK, this should work now in version 5.0.0 if you set settings.user.checkFEuser=3
Can you test it?

**Comment by gckng** [comment]
Looks great! Thank you very much, awesome support!
